Makefile

```diff
@@ -31,59 +31,51 @@ SERVER_MANAGER_TO_BIN= server-manager
 # Build settings
 DEPDIR := third-party
 SRCDIR := $(abspath src)
-BUILDBASE := $(abspath build)
-BUILDDIR = $(BUILDBASE)/$(UNAME)_$(shell uname -m)
 SUBCLEAN := $(addsuffix .clean,$(SRCDIR))
-DEPDIRS := $(addprefix $(DEPDIR)/,grpc)
-DEPCLEAN := $(addsuffix .clean,$(DEPDIRS))
 SERVER_MANAGER_PROTO := lib/grpc-interfaces/server-manager.proto
-GRPC_VERSION ?= v1.50.0
+HEALTHCHECK_PROTO := third-party/health.proto
 
 # Docker image settings
 TAG ?= devel
-EMULATOR_TAG ?= 0.14.0
+EMULATOR_TAG ?= devel
 EMULATOR_REPOSITORY ?= cartesi/machine-emulator
 
 # Mac OS X specific settings
 ifeq ($(UNAME),Darwin)
 LUA_PLAT ?= macosx
 export CC = clang
 export CXX = clang++
-LIBRARY_PATH := "export DYLD_LIBRARY_PATH=$(BUILDDIR)/lib"
 LIB_EXTENSION = dylib
-DEP_TO_LIB += *.$(LIB_EXTENSION)
 
 # Linux specific settings
 else ifeq ($(UNAME),Linux)
-LIBRARY_PATH := "export LD_LIBRARY_PATH=$(BUILDDIR)/lib:$(SRCDIR)"
 LIB_EXTENSION := so
-DEP_TO_LIB += *.$(LIB_EXTENSION)*
 # Unknown platform
 else
 LIB_EXTENSION := dll
-DEP_TO_LIB += *.$(LIB_EXTENSION)
 endif
 
-# Check if some binary dependencies already exists on build directory to skip
-# downloading and building them.
-DEPBINS := $(addprefix $(BUILDDIR)/,lib/libgrpc.$(LIB_EXTENSION))
+# Test settings
+MANAGER_ADDRESS?=127.0.0.1:5001
+FAST_TEST?=false
+export MANAGER_ADDRESS
+export FAST_TEST
 
 all: source-default
 
 env:
 	@echo $(LIBRARY_PATH)
-	@echo "export PATH='$(SRCDIR):$(BUILDDIR)/bin:/opt/cartesi/bin:${PATH}'"
-	@echo "export LUA_CPATH='./?.so;$(SRCDIR)/?.so;/opt/cartesi/lib/lua/5.3/?.so;$$(lua5.3 -e 'print(package.cpath)')'"
-	@echo "export LUA_PATH='./?.lua;$(SRCDIR)/?.lua;/opt/cartesi/share/lua/5.3/?.lua;$$(lua5.3 -e 'print(package.path)')'"
+	@echo "export PATH='$(SRCDIR):/opt/cartesi/bin:${PATH}'"
+	@echo "export LUA_CPATH_5_4='./?.so;$(SRCDIR)/?.so;/opt/cartesi/lib/lua/5.4/?.so;$$(lua5.4 -e 'print(package.cpath)')'"
+	@echo "export LUA_PATH_5_4='./?.lua;$(SRCDIR)/?.lua;/opt/cartesi/share/lua/5.4/?.lua;$$(lua5.4 -e 'print(package.path)')'"
```

Comment:
Here should do the same way latest `machine-emulator` is doing on its `env` target, using the `${LUA_PATH_5_4:-;}` trick.
Also for some systems cartesi machine will not be installed in `/opt/cartesi`, so maybe change `/opt/cartesi/` to `$(PREFIX)`

Reply:
I pushed a fixup with the change.